packages/front/svelte/views/establishment/Establishment.svelte

```diff
@@ -16,29 +16,28 @@
 
     const titles = ["Tableau de bord", "Contacts", "Pièces administratives", "Informations bancaires"];
     const associationPromise = associationService.getAssociation(siretToSiren(id));
-    const etablissementPromise = establishmentService.getBySiret(id);
+    const establishmentPromise = establishmentService.getBySiret(id);
+    const promises = Promise.all([associationPromise, establishmentPromise]);
```

Comment:
```suggestion
    const promises = Promise.all([associationPromise, establishmentPromise]).then(result => ({ association; result[0], establishment: result[1] }));
```

comme ça dans l'affichage on a des noms de variables + clairs que `result[0]` et `result[1]`

Reply:
suggestion à ne pas prendre seule ;)